src/test/java/uk/gov/justice/hmpps/prison/repository/ReferenceDataRepositoryTest.java

```diff
@@ -58,6 +60,31 @@ public void testGetReferenceCodeByDomainAndCodeWithSubCodes() {
         assertThat(refCode.get().getSubCodes()).isNotEmpty();
     }
 
+    @ParameterizedTest(name = "{index} => domain={0}, code={1}, withSubCodes={2}")
```

Comment:
Pointless test.  You're testing that the *test* reference data has been asserted correctly

Reply:
How can I make it better, please? the test verifies these codes exist and checks which ones have subcodes. I was told to look at the existing test code and follow the same style, existing tests in that file do the same type of checks, and inserting these records impacted the expectation of sorting logic results in the right way. But happy to make it better please let me know how. Thank you